Makefile

```
@@ -35,16 +35,18 @@ src/maps/%.lua: src/maps/%.tmx bin/tmx2lua
 
 bin/tmx2lua:
 	mkdir -p bin
-	$(wget) http://hawkthorne.github.com/tmx2lua/downloads/$(TMXTAR)
-	tar -xvf $(TMXTAR)
-	rm -f $(TMXTAR)
-	mv tmx2lua bin
+	$(wget) https://github.com/hawkthorne/tmx2lua/archive/master.zip
+	unzip master.zip
+	rm -rf master.zip
+	make -C tmx2lua-master
```

```suggestion
	make -C tmx2lua-master install
	make -C tmx2lua-master
```

When remaking the file, make sure that `go-tmx/tmx` is installed.